localstripe/resources.py

```diff
@@ -558,6 +558,7 @@ def __init__(self, description=None, email=None, invoice_settings=None,
         self.discount = None
         self.shipping = None
         self.default_source = None
+        self.name = name or ''
```

Comment:
If `name` is provided, it needs to be checked to be a string (like for `email` and `description` above, line 518).

Also, can you put it before `email` and `description`, for clarity? (both here and in `__init__()` arguments)